liste.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<!--
+  SPDX-License-Identifier: AGPL-3.0-only
+  SPDX-FileCopyrightText: 2025 Tim Beckmann <beckmann.tim@fh-swf.de>
+-->
+
+<html lang="de">
+
+<head>
+    <load src="partials/meta.html" />
+    <title>List | EinkaufsApp</title>
+    <link href="src/list.css" type="text/css" rel="stylesheet" />
+    <script type="module" src="/src/list.ts"></script>
+</head>
+
+<body>
+    <load src="partials/standardheader.html" pageTitle="Anmeldung" />
+    <main>
+        <div id="listInfo">
+            <input type="date" id="deadline" disabled required>
+            <input type="text" id="listTitel" disabled required>
+            <br/>
+            <textarea id="listDescription" rows="4" cols="50" disabled required></textarea>
+            <br />
+            <button type="button" id="editDescriptionButton">&#9998;</button>
+        </div>
+        <div id="listContent">
+            <table id="productTable">
+                <thead>
+                    <tr>
+                        <th></th>
+                        <th id="name">Artikelname</th>
+                        <th>Anzahl</th>
+                        <th></th>
+                        <th></th>
+                    </tr>
+                    <tr>
+                        <form id="newProduct">
+                            <th></th>
+                            <th><input type="text" id="productName" required></th>
+                            <th><input type="number" id="quantity" min="1" required></th>
```

Comment:
Please define placeholders to signal that these are meant to allow for creating new entries on the shopping list.

Something like "Produktname" and "0" maybe